pygfx/utils/show.py

```diff
@@ -0,0 +1,48 @@
+"""
+Quickly visualize world objects with as
+little boilerplate as possible.
+"""
+
+from .. import (
+    Background,
+    BackgroundMaterial,
+    WgpuRenderer,
+    WorldObject,
+    Scene,
+    PerspectiveCamera,
+    OrbitControls,
+)
+from ..linalg import Vector3
+
+
+def show(object: WorldObject):
+    from wgpu.gui.auto import WgpuCanvas, run
+
+    if isinstance(object, Scene):
+        scene = object
+    else:
+        scene = Scene()
+        scene.add(object)
+
+        background = Background(None, BackgroundMaterial((0, 1, 0, 1), (0, 1, 1, 1)))
+        scene.add(background)
+
+    camera = PerspectiveCamera(70, 16 / 9)
+
+    pos = object.get_world_position()
+    _, radius = object.get_world_bounding_sphere()
+    camera.position.copy(pos).add_scaled_vector(Vector3(1, 1, 1), radius * 1.25)
```

Comment:
should this be a util func on camera?

Reply:
Would that be the same as https://github.com/pygfx/pygfx/issues/194?

Reply:
Maybe `look_at_scene()` :)

Reply:
Yes it would!

Reply:
I ended up calling it `show_object`